recipes/flock/bld.bat

```diff
@@ -0,0 +1,3 @@
+"%R%" CMD INSTALL --build .
```

Comment:
I created the file because it says to create one in the conda tuto =)
if it doesn't have to be there i'll remove it.